data/helpers.d/filesystem

```diff
@@ -52,6 +52,9 @@ ynh_backup() {
     # ... or just copy the directory
     eval $SUDO_CMD mkdir -p $(dirname "${DESTPATH}")
     eval $SUDO_CMD cp -a "${SRCPATH}" "${DESTPATH}"
+
+    # Store the source path
+    echo "${DESTPATH} ${SRCPATH}" | tee -a backup_list > /dev/null
```

Comment:
Ok, so if you work on a new backup system. I think this helper should be follow your work.

How can we use this list with your PR?